### PR TITLE
Change radio button on edit corrective action

### PR DESCRIPTION
**Pull request by KyleMacPherson**
https://trello.com/c/qk5MQogL/688-update-edit-corrective-action-behaviour-and-activity-log-file-upload-removal

## Description
It seems the only remaining work that needed to be done on the ticket above was to dynamically render the text for the no option on the question "Are there any files related to the action?" to say "Remove attached file" if there is an existing file attached to the corrective action being edited.

## Checklist:
- [ ] Have you documented your changes in the pull request description?
- [ ] Does the change present any security considerations?
- [ ] Is any gem functionality overloaded? Eg: Devise controller methods being overloaded.
- [ ] Has acceptance criteria been tested by a peer?
- [ ] Has the CHANGELOG been updated? (If change is worth telling users about.)

### General testing (author)
- [ ] Test without JavaScript
- [ ] Test on small screen

### Accessibility testing (author)
- [ ] Works keyboard only
- [ ] Tested with one screen reader
- [ ] Zoom page to 400% - content still visible
- [ ] Disable CSS - does content make sense and appear in a logical order?

### CODE REVIEW

**Comment by nasirkhanbeis** [comment]
spotted minor issue, when same file is attached then audit log shows empty log record. Expected not to show any changes when replaced with same file.

**Review by nasirkhanbeis — APPROVED** [review]
Tested this on review app, finally appears all good. Nice work @KyleMacPherson 
there are some further improvements needed which I will raise as separate ticket.

File: app/services/update_corrective_action.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@ def call
     corrective_action.transaction do
       corrective_action.document.detach unless related_file
       replace_attached_file             if file_changed?
+      remove_blanks_from_changes
```
Comment (slorek) [inline app/services/update_corrective_action.rb:15]: I think this should go in the form class as it is sanitising user input. We have a helper called `SanitizationHelper` which can do this. An example of usage is here https://github.com/UKGovernmentBEIS/beis-opss-psd/blob/master/app/forms/corrective_action_form.rb#L54

File: app/views/investigations/activities/corrective_action/_update.erb [hunk 2]
```diff
@@ -29,7 +29,11 @@
   <% end %>
 
   <% if activity.business_updated? %>
```
Comment (slorek) [inline app/views/investigations/activities/corrective_action/_update.erb:31]: Not a big one, but as a rule we should be reading the `business_id` in the `metadata` property. The extra relationships on activity models were added a long time ago to patch over an inflexible model, and will ideally be removed in the near future.

**Review by slorek — CHANGES_REQUESTED** [review]

File: app/services/update_corrective_action.rb (hunk 1 above)
Comment (KyleMacPherson) [reply app/services/update_corrective_action.rb]: This example usage is actually in place here, the issue is that we were setting this value to nil via the sanitisation (and also via a method in the update_corrective_action class, but the record on creation is being saved with the file description value as an empty string. I have made some changes to make this uniform across both create and update and removed any sanitisation being done in the update_corrective_action class.

Let me know what you think!

**Review by slorek — APPROVED** [review]

**Pull request merged**